lib/open_api_spex/plug/cache.ex

```diff
@@ -50,7 +50,7 @@ defmodule OpenApiSpex.Plug.Cache do
   @spec refresh() :: :ok
   def refresh do
     adapter = adapter()
-    adapter.erase()
+    adapter.erase(adapter)
```

Comment:
I think this will stop it from crashing, but not actually clear the cache.

The refresh function should take a `spec_module` argument and pass it through to adapter, or alternatively we should add an `erase_all` callback to the adapter behaviour that doesn't require the spec module to be given.